_pages/business-units/18f/chapters/engineering.md

```diff
@@ -73,17 +73,17 @@ See [the org chart]({{site.baseurl}}/org-chart).
 
 ### What is expected of me regarding time tracking and how I spend my time?
 
-The default expectation for 18F engineers is to be committed to a single project full-time, which means 32 hours a week. If you'd like a different arrangement, for example to work on multiple projects part-time or something else, please speak with your facilitator.
+The default expectation for 18F engineers is to be committed to a single project full-time, which means a minimum of 32 hours a week (billing more than 32 hours a week is okay). If you'd like a different arrangement, for example to work on multiple projects part-time or something else, please speak with your facilitator.
 
-Your project commitments should be captured by the [#staffing-resourcing](https://gsa-tts.slack.com/messages/staffing-resourcing/) team. Your remaining time is intended for general meetings, like the weekly all hands, and working group or guild participation. If you don’t have enough project work to fill 32 hours, please let your facilitator know.
+Your project commitments should be captured by the [#18f-staffing](https://gsa-tts.slack.com/messages/18f-staffing/) team. Your remaining time (up to 8 hours) is intended for general meetings, like the weekly all hands, and working group or guild participation. If you don’t have enough project work to fill at least 32 hours, please let your facilitator know.
 
 All 18F staff track their time in [Tock](https://tock.18f.gov/). Learn more in the [Tock guide]({{site.baseurl}}/tock/). While cost recovery is an important organizational goal, internal non-billable projects are important as well. Engineers needn't worry about how many billable hours they work, but rather how many project hours.
```

Comment:
` Engineers needn't worry about how many billable hours they work, but rather how many project hours.`

I have no idea what this means. Project hours and billable hours aren't the same?

Reply:
That sentence makes no sense. Let's rip it out.